src/Filters/Filters.jl

```diff
@@ -1,18 +1,15 @@
 module Filters
-using ..DSP: @importffts, mul!, rmul!
+using ..DSP: mul!, rmul!
```

Comment:
More obvious if these are used from `LinearAlgebra`, i.e. added to the line below instead.